docs/zh_cn/inference/vl_pipeline.md

```diff
@@ -97,6 +97,22 @@ response = pipe(('describe this image', image), gen_config=gen_config)
 print(response)
 ```
 
+### 自定义图片 token 的位置
+
+默认情况下，LMDeploy 会根据算法 repo 提供的对话模版将表示图片的特殊 token 插入到 user prompt 中，但在一些模型中，图片 token 的位置并没有限制，如 deepseek-vl，或者用户需要自定义图片 token 插入的位置。这种情况下，用户需要手动将表示图片的 token 插入到 prompt 中。
+
+```python
+from lmdeploy import pipeline
+from lmdeploy.vl import load_image
+from lmdeploy.vl.constants import IMAGE_TOKEN
+
+pipe = pipeline('deepseek-ai/deepseek-vl-7b-chat')
```

Comment:
Shall we try `deepseek-ai/deepseek-vl-1.3b-chat` instead as an example? It is smaller and can be downloaded faster.